build.go

```diff
@@ -43,16 +43,33 @@ func buildStaticSite(dest string) error {
 		log.Printf("error while processing root path, err: %s", err.Error())
 	}
 
-	EachPage(context.Background(), func(p Page) {
-		err := buildRoute(
+	// building 404 page
+	if NOT_FOUND_PAGE != "" {
```

Comment:
instead of rendering 404.md manually then exclude it in line 62. How about we just let it render as usual then copy it from `404/index.html` to `404.html` and in this case it'll be simpler and we'll have both `/404` and `404.html` in our site.

Reply:
Done, now if we render /dest/NOT_FOUND_PAGE/index.html we copy it to /dest/404.html.